keyboards/satan/keymaps/poker/keymap.c

```diff
@@ -89,19 +93,13 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   ),
 };
 
-enum function_id {
-    SHIFT_ESC,
-};
+bool process_record_user(uint16_t keycode, keyrecord_t* record) {
+  static uint8_t shift_esc_shift_mask;
 
-const uint16_t PROGMEM fn_actions[] = {
-  [0]  = ACTION_FUNCTION(SHIFT_ESC),
-};
+  switch (keycode) {
+    case SFT_ESC:
+      shift_esc_shift_mask = get_mods() & MODS_CTRL_MASK;
 
-void action_function(keyrecord_t *record, uint8_t id, uint8_t opt) {
-  static uint8_t shift_esc_shift_mask;
-  switch (id) {
-    case SHIFT_ESC:
-      shift_esc_shift_mask = get_mods()&MODS_CTRL_MASK;
       if (record->event.pressed) {
         if (shift_esc_shift_mask) {
```

Comment:
```suggestion
        if (get_mods() & MODS_SHIFT_MASK) {
```
Same changes as above.